src/actions/variable.ts

```diff
@@ -8,8 +8,32 @@ import {ExpressionQualifier} from "../qualifiers/expression/ExpressionQualifier"
  * Defines a new user variable with the given value.
  * @memberOf Actions
  * @namespace Variable
+ * @example
+ * import {Cloudinary} from "@cloudinary/base/instance/Cloudinary";
+ * import {
+ *  set,
+ *  setAssetReference,
+ *  setFromContext,
+ *  setFromMetadata
+ * } from "@cloudinary/base/actions/variable";
+ * import {scale} from "@cloudinary/base/actions/resize";
+ *
+ * const yourCldInstance = new Cloudinary({cloud:{cloudName:'demo'}});
+ *
+ * const image1 = yourCldInstance.image('woman');
```

Comment:
@patrick-tolosa we can make this shorter - no need to define multiple images to show usage

Reply:
Done